tests/ttnn/unit_tests/operations/test_moreh_fold.py

```diff
@@ -0,0 +1,108 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
```

Comment:
As you have compute kernel, you should add test for compute kernel config.

Reply:
Resolved in last commit. I have removed compute kernel config as this op doesn't require compute kernel.